Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -16,6 +16,54 @@
 
 using namespace amrex;
 
```

Comment:
The code below has been moved from the file `WarpXFFT.cpp`, this is because this code should still be compiled on GPU (local FFT solver), but the rest of the contents of `WarpXFFT.cpp` does not need to be compiled on GPU.